src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Conditions/Condition.cs

```diff
@@ -15,29 +15,29 @@ public enum Condition
     {
         [DisplayName("Contains")]
         Contains,
-        [DisplayName("StartsWith")]
```

Comment:
We just updated the resources (they were wrong). We have accepted your pr and will update the resx. Grab the latest and you'll see the magic :)

Thanks for the PR, well done :+1: